x-pack/plugins/apm/public/components/app/TransactionDetails/WaterfallWithSummmary/CopyTraceLink.tsx

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import {
+  EuiButtonIcon,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiText,
+  EuiToolTip,
+} from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+import React, { useCallback } from 'react';
+import { useApmPluginContext } from '../../../../hooks/useApmPluginContext';
+import { useKibana } from '../../../../../../../../src/plugins/kibana_react/public';
+import { Transaction as ITransaction } from '../../../../../typings/es_schemas/ui/transaction';
```

Comment:
Unless there is a naming collision I don't think this needs to be renamed. We're trying to stop using the `I` prefix for interfaces.